multiaddr/codecs/ip4.py

```diff
@@ -1,6 +1,6 @@
 import netaddr
-from ..codecs import CodecBase
 
+from ..codecs import CodecBase
```

Comment:
Was was a empty line added? 
To improve readability ?
What's the general rule?